src/components/projects/project-edit-form/project-edit-container/project-edit-details/project-edit-details.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Project } from 'utils';
+import {
+  weeksPipe,
+  wbsPipe,
+  endDatePipe,
+  fullNamePipe,
+  listPipe
+} from '../../../../../shared/pipes';
+import PageBlock from '../../../../shared/page-block/page-block';
+import { Col, Container, Row, Form } from 'react-bootstrap';
+// import './work-package-details.module.css';
+import { EditModeContext } from '../project-edit-container';
+import { useContext } from 'react';
+import ProjectFormDetail from './project-form-detail/project-form-detail';
+
+// new parts added at the bottom
+interface projectDetailsProps {
+    project: Project;
+}
+
+const ProjectEditDetails: React.FC<projectDetailsProps> = ({ project }) => {
+    const editMode = useContext(EditModeContext);
+    const detailsBody = (
+      <Container fluid>
+        <Form>
+          <Row>
+            <b>Project Details</b>
+            <Form.Group>
+              <Form.Control>
+                placeholder={"status"}
+                option={"inactive"}
+                option={"active"}
+              </Form.Control>
+            </Form.Group>
+          </Row>
+          <Row>
+            <Col xs={12} md={6}> 
+              <ProjectFormDetail type="title" isReadonly={false} details={project} />
+              <ProjectFormDetail type="wbs" isReadonly={false} details={project} />
+              <ProjectFormDetail type="project-lead"  isReadonly={false} details={project} />
+              <ProjectFormDetail type="project-manager" isReadonly={false} details={project} />
+            </Col>
+            <Col xs={6} md={4}>
+              <ProjectFormDetail type="duration" isReadonly={true} details={project} />
+              <ProjectFormDetail type="start-date" isReadonly={true} details={project} />
+              <ProjectFormDetail type="end-date" isReadonly={true} details={project} />
+              <br />
+              <ProjectFormDetail type="progress" isReadonly={false} details={project} />
```

Comment:
I explained this to Zack, but basically when you create a component that does multiple different things, it's much preferable to have the differences be passed in as parameters, rather than storing the different options inside the component and selecting which option to display. In some ways this is like from fundies 1 with abstracting over multiple functions (and passing in the different parts as parameters) vs a function with a cond inside that decides which case to return. Making it a abstracted component would get you going in the direction where the reusable component could be used for much more beyond this page